python/hail/dataset.py

```diff
@@ -356,7 +356,9 @@ def annotate_samples_expr(self, expr):
         Compute per-sample GQ statistics for hets:
 
         >>> vds_result = (vds.annotate_samples_expr('sa.gqHetStats = gs.filter(g => g.isHet()).map(g => g.gq).stats()')
-        ...     .export_samples('output/samples.txt', 'sample = s, het_gq_mean = sa.gqHetStats.mean'))
+        ...                  .samples_table()
+        ...                  .select('sample = s', 'het_gq_mean = sa.gqHetStats.mean')
```

Comment:
Does this need to get changed back to .select(['sample = s', 'het_gq_mean = sa.gqHetStats.mean']) for the vararg/kwarg fix? Docs are failing here.